src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -650,6 +659,36 @@ internal PropertyMirror(ProcedureNode procNode, bool isSetter = false)
                 this.Name = PropertyName;
                 this.isSetter = isSetter;
             }
+
+            private string Argument
```

Comment:
would you mind adding a comment for what this is supposed to produce?

Reply:
The static getter equivalent of a property will have exactly one argument.

Reply:
makes sense.